packages/entando-ddtable/sass/patternfly.scss

```diff
@@ -0,0 +1,6 @@
+@import "../../node_modules/patternfly/dist/sass/patternfly/variables";
```

Comment:
Patternfly should be declared in dependencies. or in peerDependencies

Reply:
added it to the dev dependencies: it is only used on pre-publish